static/templates/settings/admin_linkifier_list.hbs

```diff
@@ -11,6 +11,9 @@
         <button class="button small delete btn-danger" data-linkifier-id="{{id}}" title="{{t "Delete" }}">
             <i class="fa fa-trash-o"></i>
         </button>
+        <button class="button small edit btn-warning" data-linkifier-id="{{id}}" title="{{t "Edit" }}">
```

Comment:
I fixed several issues here:
* Missing aria-label, which should always be present alongside title.
* Invalid quoting within the `t` tag -- you need to use a different quote type.
* The edit button should be before the delete button.

I also notice there's a weird margin within the pencil icons we need to fix; will attempt a new commit to fix that.

Reply:
Although the quoting looks weird, it isn’t actually a problem. `{{t "Edit" }}` is evaluated by Handlebars well before the outer quotes are parsed as HTML.

Reply:
Yeah, but it confuses one's editor (etc.), and thus seemed worth fixing.